include/clspv/Option.h

```diff
@@ -15,6 +15,8 @@
 #ifndef CLSPV_INCLUDE_CLSPV_OPTION_H_
 #define CLSPV_INCLUDE_CLSPV_OPTION_H_
 
+#include <cstdint>
```

Comment:
Noticed this was an issue while debugging and including this file in UndoBoolPass.